symfony/framework-bundle/4.2/src/bootstrap.php

```diff
@@ -4,8 +4,14 @@
 
 use Symfony\Component\Dotenv\Dotenv;
 
-if (class_exists(Dotenv::class)) {
-    (new Dotenv())->loadForEnv($_SERVER['APP_ENV'] ?? $_ENV['APP_ENV'] ?? 'dev', __DIR__.'/../.env');
-} elseif (!isset($_SERVER['APP_ENV']) && !isset($_ENV['APP_ENV'])) {
+$envFromEnv = $_SERVER['APP_ENV'] ?? $_ENV['APP_ENV'] ?? null;
+$env = $envFromEnv ?? 'dev';
+$prod = 'prod' === $env;
+
+if (!$env && class_exists(Dotenv::class)) {
```

Comment:
`$env` cannot be null at this point. Did you mean `$prod` ?

Reply:
indeed, fixed